lib/tasks/migrate_data.rake

```diff
@@ -0,0 +1,18 @@
+def create_commits!(repository)
+  repository.git.commits { |c| repository.commit_for!(c.oid) }
+end
+
+def attach_commit!(ontology_version)
+  commit = ontology_version.repository.commits.
+    find_by_commit_oid(ontology_version.commit_oid)
```

Comment:
Just wondering, why the dot is at the end of the line instead of right at the beginning of the called object method?
Without it, it looks like a "static"/"target unrelated"/"dangling" call - IMHO a little bit confusing/harder to read (at least in Java it is a common coding rule to start a line continuation with a dot after the indent if possible - not sure, whether ruby allows that at all).